spec/models/restitution/inventaire_spec.rb

```diff
@@ -235,4 +235,27 @@
                                                    Competence::ORGANISATION_METHODE])
     end
   end
+
+  describe '#version?' do
+    context "si il y a un numéro de version dans l'évènement démarrage" do
+      it 'compare le numéro de version de la situation' do
+        evenements = [
+          build(:evenement_demarrage, donnees: { version: '2' })
+        ]
+        restitution = described_class.new(campagne, evenements)
+        expect(restitution.version?('2')).to be true
+        expect(restitution.version?('1')).to be false
+      end
```

Comment:
```suggestion
    before do
      evenements = [
        build(:evenement_demarrage, donnees: { version: '2' })
      ]
      restitution = described_class.new(campagne, evenements)
    end
    it { expect(restitution.version?('2')).to be true }
    it { expect(restitution.version?('1')).to be false }
```

Pour respecter cette norme : https://www.betterspecs.org/#single